Makefile

```
@@ -0,0 +1,248 @@
+# Smart Pen Project Makefile
+# ===========================
+
+.PHONY: help setup install start dev stop clean logs test build deploy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)